src/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -367,6 +370,14 @@ private boolean scanSingleNode(Plugin plugin, StructuralNode node) {
 
             test = plugin.getClass().newInstance();
             test.setConfig(plugin.getConfig());
+            // Set the configuration rules
+            FileConfiguration confFile = Model.getSingleton().getOptionsParam().getConfig();

```

Comment:
Could this (RuleConfigParam) be passed as parameter of HostProcess instead? (It would be easier to use custom configurations per scan, for example, through an advanced tab in Active Scan dialogue.)